File: OpenApi2.0/push-openapi-php-demo/QueryPushStatByApp.php
<?php

include_once '../aliyun-php-sdk-core/Config.php';
include 'Push/Request/V20160801/QueryPushStatByAppRequest.php';
use \Push\Request\V20160801 as Push;

$accessKeyId = "your accessKeyId";
$accessKeySecret = "your accessKeySecret";
$appKey = "your appKey";

$iClientProfile = DefaultProfile::getProfile("cn-hangzhou", $accessKeyId, $accessKeySecret);
$client = new DefaultAcsClient($iClientProfile);
$request = new Push\QueryPushStatByAppRequest();

$request->setAppKey($appKey);
//DAY: 天粒度
$request->setGranularity("DAY");
//查询近七天的数据
$startTime =  gmdate('Y-m-d\TH:i:s\Z', strtotime('-7 day'));
$endTime = gmdate('Y-m-d\TH:i:s\Z', time());
$request->setStartTime($startTime);
$request->setEndTime($endTime);

$response = $client->getAcsResponse($request);
print_r("\r\n");
print_r($response);

?>

File: OpenApi2.0/push-openapi-python-demo/QueryPushStatByApp.py
#!/usr/bin/python
#coding=utf-8
from datetime import *
import time
import properties
from aliyunsdkpush.request.v20160801 import QueryPushStatByAppRequest
from aliyunsdkcore import client

clt = client.AcsClient(properties.accessKeyId,properties.accessKeySecret,properties.regionId)

request = QueryPushStatByAppRequest.QueryPushStatByAppRequest()
request.set_AppKey(properties.appKey)

## 按天查询
request.set_Granularity('DAY')

## 查询近三天的数据
startDate = datetime.utcnow() + timedelta(days = -3)
endDate = datetime.utcnow()

## 转换成ISO8601T数据格式
startTime = startDate.strftime("%Y-%m-%dT%XZ")
endTime = endDate.strftime("%Y-%m-%dT%XZ")

request.set_StartTime(startTime)
request.set_EndTime(endTime)

result = clt.do_action(request)
print result
